lib/kube/proxy/server.go

```diff
@@ -164,11 +207,21 @@ func (t *TLSServer) GetServerInfo() (services.Server, error) {
 		addr = t.listener.Addr().String()
 	}
 
+	// Both proxy and kubernetes services can run in the same instance (same
+	// ServerID). Add a name suffix to make them distinct.
+	//
+	// Note: we *don't* want to add suffix for kubernetes_service!
+	// This breaks reverse tunnel routing, which uses server.Name.
+	name := t.ServerID
+	if !t.NewKubeService {
+		name += "/proxy_service"
```

Comment:
Nit: Not sure if this name is used anywhere but I wonder if an underscore or a dash instead of a slash would be a safer choice for a suffix in case this name is gonna be used in cli/path somewhere.

Reply:
Updated to `-proxy_service`.